modules/cluster-logging-rn-5.3.13.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-13_{context}"]
+= Logging 5.3.13
+This release includes link:https://access.redhat.com/errata/RHSA-2022:6882[RHSA-2022:68828-OpenShift Logging Bug Fix Release 5.3.13].
+
+[id="openshift-logging-5-3-13-bug-fixes"]
+== Bug fixes
+None.
+
+[id="openshift-logging-5-3-13-CVEs"]
+== CVEs
+.Click to expand CVEs
```

Comment:
In the _preview_, these CVEs are expanded by default. If you intend them to be collapsed, there might be a configuration setting to make that happen. Otherwise, would it make sense to change this phrase to something like:
```suggestion
.Click to collapse the list of CVEs
```